dynawo/sources/Models/Modelica/Dynawo/Electrical/Machines/OmegaRef/GeneratorPVDiagramPQ.mo

```diff
@@ -29,6 +29,7 @@ model GeneratorPVDiagramPQ "Generator with active power / frequency regulation a
                               AbsorptionMax "Reactive power is fixed to its absorption limit",
                               GenerationMax "Reactive power is fixed to its generation limit");
 
+  Connectors.ImPin deltaURefPu(value(start = 0)) "Voltage regulation step in pu (base UNom)";
```

Comment:
Is that always a step ? If not, I would write "Additional voltage reference in pu (base UNom)" .